### PR TITLE
seafile-client: Use qt5's mkDerivation

### PR DESCRIPTION
**Pull request by greizgh**
###### Motivation for this change

Following #65399 this update seafile-client to use qt5's mkDerivation.

Launching seafile-applet from systemd user unit was not working, with these changes it's working again.
This is my first contribution to nixpkg, feel free to comment and suggest improvements.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/networking/seafile-client/default.nix [hunk 1]
```diff
@@ -15,17 +15,14 @@ stdenv.mkDerivation rec {
...
     ++ optional withShibboleth qtwebengine;
 
   cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ]
     ++ optional withShibboleth "-DBUILD_SHIBBOLETH_SUPPORT=ON";
 
-  postInstall = ''
-    wrapProgram $out/bin/seafile-applet \
-      --suffix PATH : ${stdenv.lib.makeBinPath [ ccnet seafile-shared ]}
-  '';
+  qtWrapperArgs = [ ''--suffix PATH : ${stdenv.lib.makeBinPath [ ccnet seafile-shared ]}''];
```
Comment (worldofpeace) [inline pkgs/applications/networking/seafile-client/default.nix]:
```suggestion
  qtWrapperArgs = [ 
  "--suffix PATH : ${stdenv.lib.makeBinPath [ ccnet seafile-shared ]}"
  ];
```

**Comment by worldofpeace** [comment]
Can you link the tracking issue in the commit body @greizgh?

**Comment by greizgh** [comment]
> Can you link the tracking issue in the commit body @greizgh?

Sure thing! Done and formatted the wrapper args

**Comment by worldofpeace** [comment]
Thanks for fixing this @greizgh :sparkles:

**Pull request merged**